groups/bdl/bdlb/bdlb_guidutil.h

```diff
@@ -217,9 +220,9 @@ struct GuidUtil {
         // 'variant' bits set to '10' and four 'version' bits set to '0100'.
         // The behavior is undefined unless 'result' refers to a contiguous
         // sequence of at least 'numGuids' Guid objects.  Note that this
-        // function uses the PCG random engine to generate high quality, albeit
-        // not cryptographically secure, random numbers for GUIDs. This method
-        // is typically faster than generating cryptographically secure GUIDs.
+        // function generates high quality, albeit not cryptographically
+        // secure, random numbers for GUIDs. This method is typically faster
+        // than generating cryptographically secure GUIDs.
```

Comment:
Remove the last sentence - it doesn't refer to this function only, thus this information is in the component doc.  Also please update the rest of `generateNonSecure` function doc to match.